README.md

```diff
@@ -15,20 +15,57 @@
 |:----------------:|:----------------------------:|
 | v0.1.*           | v1.5.*                       |
 
-## Installation
```

Comment:
you should replace "[Experimental] Starter kit to quickly setup your own Akeneo PIM Extension" by a sentence explaining the purpose of this tool, aka "allow you to quickly create your own PIM Extension" + maybe a link to the marketplace.